modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.html

```diff
@@ -10,7 +10,7 @@
 			/]
```

Comment:
Please rename this commit to _LPS-176972 Adjustment position dropdown_

Reply:
Done!